src/i18n/fr_FR/buildings.ts

```diff
@@ -298,6 +340,35 @@ export default {
                 },
                 9
             ),
+            {
+                caption: 'Grand poste de police',
+                credits: 1_000_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[6]?.length ?? 0) / 10),
```

Comment:
In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.